vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -176,7 +176,13 @@ namespace Microsoft.ReactNative
       "If this is not provided, the value of @.JavaScriptBundleFile is used.")
     String DebugBundlePath { get; set; };
 
-    DOC_STRING("Base path used for the location of the bundle.")
+    DOC_STRING(
```

Comment:
I feel like this would be more useful with some example.  The format is `resource://moduleName/resourceID` or `resource://resourceID`, right?  That isn't clear from this description.

Reply:
added examples